Lib/pathlib.py

```diff
@@ -1166,6 +1189,210 @@ def walk(self, top_down=True, on_error=None, follow_symlinks=False):
 
             paths += [path._make_child_relpath(d) for d in reversed(dirnames)]
 
+    def absolute(self):
+        """Return an absolute version of this path by prepending the current
+        working directory. No normalization or symlink resolution is performed.
```

Comment:
For a more accurate docstring (virtual file systems may have no notion of a CWD)
```suggestion
        """Return an absolute version of this path
        No normalization or symlink resolution is performed.
```